code/modules/reagents/reagents/food-Drinks.dm

```diff
@@ -2937,3 +2956,201 @@
 	glass_icon_state = "berryjuice"
 	glass_name = "strawberry juice"
 	glass_desc = "Sweet and sugary, but also very refreshing!"
+
+//SoJ based drinks
+
+
+/datum/reagent/ethanol/southern_saki
+	name = "Southern Sake"
+	id = "southern_saki"
+	description = "Best served warm and in a shallow bowl for lapping."
+	taste_description = "unbearable bitterness"
+	color = "#F9F9F9" // rgb(98, 98, 98)
+	strength = 50 //rice wine stronk
+
+	glass_unique_appearance = TRUE
+	glass_icon_state = "sake"
+	glass_name = "Southern Sake"
+	glass_desc = "Best served warm and in a shallow bowl for lapping."
+
+/datum/reagent/ethanol/nothern_mead
+	name = "Nothern Mead"
+	id = "nothern_mead"
+	description = "Said to make even the strongest of accents all the more unintelligible."
+	taste_description = "robust hops and carbon"
+	color = "#E0CE8A" // rgb(88, 81, 54)
+	strength = 140 //stronk
```

Comment:
_Ackshually_, `strength` works inversely as tg's `boozepwr` code wise: [The lower the value, the stronger the drink](https://github.com/sojourn-13/sojourn-station/blob/716e363a731bef79140d21c84a4447f1fea36800/code/modules/reagents/reagents/dispenser.dm#L111), hence why Absinthe and Moonshine will get you drunker easier. If you want it to be strong, make it between 10 (baseline ethanol) and 30.